### ISSUE TITLE
[USABILITY]: 526 V2 error handling when users press Continue button should be reviewed

### ISSUE THREAD

**Issue by aeltarhoni** [department-of-veterans-affairs/va.gov-team#1705]
@1Copenut commented on [Wed Aug 22 2018](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12581)

## Issue

The error handling when users press the `Continue` button was flagged by the VA 508 office during an audit. They said the expected behavior was to focus the first form field in error when users press Continue and one or more fields have errors.

I checked this behavior on other forms, and it seems consistent that we require users to press `TAB` to move from the form to the first input in error on other forms, so it seems our behavior on the 526 disability increase form is consistent. I am putting this issue in as a way to review that behavior and start a larger conversation if necessary.

## Environment

* Windows 7 64 Bit Enterprise Edition SP1
* Google Chrome
* JAWS 2018.0

---

![picture4](https://user-images.githubusercontent.com/934879/44502630-58b30a80-a660-11e8-9d50-4794804b92d8.png)

---

@1Copenut commented on [Mon Sep 24 2018](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12581#issuecomment-424032553)

@djmunoz1 I logged an issue with the US Forms Library repo ( https://github.com/usds/us-forms-system/issues/278 ) about error handling, that seems to line up with this issue. Please advise if you'd like to leave this open, or close it and manage another way.

**Comment by Mottie** [comment]
Slack response from Trevor Pierce

> Hi team.
I reviewed the ticket and this is standard behavior on all of our forms, to require users to press <kbd>TAB</kbd> after pressing Continue to get to the first invalid required field. I reached out to the VA 508 team and referenced the new ticket to get a final conclusion.
I’ll set a todo to circle back with them next week and let you know.
No action is required until we have guidance.

**Comment by YanaRoy** [comment]
@1Copenut following up: is this still blocked?

**Comment by 1Copenut** [comment]
@YanaRoy Yes, this is a wider issue that would affect all of our forms. What do you think about closing this item for the 526 specifically and opening a new issue ticket for exploration? It might be better served that way, because any change will be sitewide.

**Comment by Mottie** [comment]
Closed. See #2533 which broadens the scope of this issue.
